charts/incubator/automatic-ripping-machine/values.yaml

```diff
@@ -0,0 +1,42 @@
+image:
+  repository: tccr.io/truecharts/automatic-ripping-machine
+  pullPolicy: IfNotPresent
+  tag: v2.6.60@sha256:80794ea2bcf0372856f83c980ee65f26eacd3253b9c97215e686fb85e2df969b
+
+securityContext:
+  container:
+    runAsNonRoot: false
+    readOnlyRootFilesystem: false
+    runAsUser: 0
+    runAsGroup: 0
+    allowPrivilegeEscalation: true
```

Comment:
Please add a warning in the docs.
Basically this makes it "GOD MODE" on the host